tests/ctl_util.py

```diff
@@ -1,20 +1,27 @@
 from typing import TYPE_CHECKING
 
 from grouper.ctl.main import main
-from tests.path_util import src_path
+from tests.path_util import db_url, src_path
 
 if TYPE_CHECKING:
     from grouper.models.base.session import Session
+    from py.path import LocalPath
     from tests.setup import SetupTest
 
 
-def call_main(session, *args):
-    # type: (Session, *str) -> None
-    argv = ["grouper-ctl", "-c", src_path("config", "dev.yaml")] + list(args)
+def call_main(session, tmpdir, *args):
+    # type: (Session, LocalPath, *str) -> None
+    argv = ["grouper-ctl", "-c", src_path("config", "dev.yaml"), "-d", db_url(tmpdir)] + list(args)
     main(sys_argv=argv, session=session)
 
 
 def run_ctl(setup, *args):
```

Comment:
To confirm I'm understanding correctly -- `run_ctl` is the new way of testing that we're slowly migrating to, right? If so, could you throw a comment on `call_main` saying essentially "this isn't what you want"? :)

Reply:
Will do.